collector/project.go

```diff
@@ -106,6 +121,42 @@ func (p Project) Collect(c *CollectorOpts) interface{} {
 			p.Pod.Update(totalPo)
 			poUtils = append(poUtils, float64(totalPo))
 		}
+
+		// Apps
+		if len(globalRancherLibraryNames) > 0 {
+			duplicateLibraryNames := false
+			for _, catalogName := range globalRancherLibraryNames {
+				// Make sure rancher library names aren't duped in this project or cluster
+				pCatalog, err := c.Client.ProjectCatalog.ByID(
+					fmt.Sprintf("%s:%s", projectID, catalogName))
+				if err == nil && pCatalog != nil {
+					duplicateLibraryNames = true
+					break
+				}
```

Comment:
maybe make sure it was a not found error?

Reply:
We'd want to error on side of not counting though. If it was an unrelated error like a connection issue, then we should assume it does exist, or just error out entirely.